install/Create_System_Configuration_Using_Cluster_Discovery_Service.md

```diff
@@ -0,0 +1,400 @@
+
+      ```bash
+      pdsh -w^nodelist -x localhost "sh "${CVT_PATH}"/scripts/create_bond0.sh"
+      ```
+
+   1. Ensure that `bond0` network interface is created on all the nodes listed in the `nodelist`.
+
+      Example output:
+
+      ![Bond0 Network Interface](../img/install/create_bond0.png)
+
+1. (`pit#`) Verify the status and list the IP addresses of the fabric switches.
+
+   ```bash
+   "${CVT_PATH}"/cds/discover_status.py fabric --username fabric_username --password fabric_password --out
+   ```
+
+1. (`pit#`) Verify the status and list the IP addresses of the PDUs.
+
+   ```bash
+   "${CVT_PATH}"/cds/discover_status.py pdu --username pdu_username --password pdu_password --out
+   ```
+
+1. (`pit#`) Verify the status and list the IP addresses of the CMCs.
+
+   ```bash
+   "${CVT_PATH}"/cds/discover_status.py cmc --username cmc_username --password cmc_password --out
+   ```
+
+### 3.4 Collect and compare the inventory data
+
+#### 3.4.1 Dynamic Data Collection
+
+1. (`pit#`) Copy the SSH key to setup passwordless-SSH for the local host.
+
+   ```bash
+   ssh-copy-id localhost
+   ```
+
+1. (`pit#`) Collect the hardware inventory data.
+
+   ```bash
+   "${CVT_PATH}"/system_inventory.py --nodefile nodelist
+   ```
+
+1. (`pit#`) Collect the fabric inventory data.
+
+   ```bash
+   "${CVT_PATH}"/fabric_inventory.py --switchfile ./cds/fabricswlist --username switch_username --password switch_password
+   ```
+
+1. (`pit#`) Collect the management network inventory.
+
+   1. Manually, create a `mgmtswlist` file in the working directory listing the IP addresses of management switches.
+
+   2. (`pit#`) Collect the management network inventory data.
+
+      ```bash
+      "${CVT_PATH}"/management_inventory.py --switchfile mgmtswlist --password password --username username
+      ```
+
+1. (`pit#`) Collect the PDU inventory data.
+
+   ```bash
+   "${CVT_PATH}"/cds/pdu_cmc_inventory.py --pdu_file pdulist
+   ```
+
+1. (`pit#`) Collect the CMC inventory data.
+
+   ```bash
+   "${CVT_PATH}"/cds/pdu_cmc_inventory.py --cmc_file cmclist
+   ```
+
+#### 3.4.2 SHCD data population
+
+1. Copy the SHCD files to the present working directory.
+
+1. (`pit#`) Create a JSON file using the CANU tool.
+
+   See [Validate the SHCD](../operations/network/management_network/validate_shcd.md#validate-the-shcd).
+
+   Example command:
+
+   ```bash
+   canu validate shcd --shcd SHCD.xlsx --architecture V1 --tabs 40G_10G,NMN,HMN --corners I12,Q38,I13,Q21,J20,U38 --json --out cabling.json
+   ```
+
+1. (`pit#`) Store the SHCD data in the CVT database.
+
+   ```bash
+   "${CVT_PATH}"/parse_shcd.py --canu_json_file cabling.json
+   ```
+
+#### 3.4.3 Server classification
+
+1. (`pit#`) Classify the servers.
+
+   > In the following command, replace `<N>` with the number of worker NCNs in the system.
+
+   ```bash
+   "${CVT_PATH}"/scripts/server_classification.py --no_of_workers <N>
+   ```
```

Comment:
Sections with a single step should not have that step enumerated.
````suggestion
(`pit#`) Classify the servers.

> In the following command, replace `<N>` with the number of worker NCNs in the system.

```bash
"${CVT_PATH}"/scripts/server_classification.py --no_of_workers <N>
```
````